third_party/txt/src/utils/LinuxUtils.h

```diff
@@ -39,6 +39,4 @@
   FRIEND_TEST_LINUX_ONLY_EXPANDED(SUITE, DISABLE_TEST_LINUX(TEST_NAME))
 #endif  // defined(__linux__)
 
-#define LINUX_ONLY_DISABLED(TEST_NAME) DISABLED_##TEST_NAME
-
```

Comment:
We might as well keep this.  If we needed it once we may well need it in the future.